app/utils/wikiQuery.tsx

```diff
@@ -76,3 +76,112 @@ export function isPartialContestantData(contestantRowData: ITableRowData): boole
     return (contestantRowData.name == null || contestantRowData.name === "") && (!contestantRowData.name2 || !contestantRowData.col2)
 }
 
+interface BBHouseGuest {
+    teamName: string
+    relationship: string
+    isParticipating: boolean
+    exitedDay: number
+}
+
+export function getCompetingEntityList(contestantData :ITableRowData[]): any {
+
+    const contestants: BBHouseGuest[] = []
+    let previousExitDay: number = 0
+
+    contestantData.forEach((element, index) => {
+
+        if (isPartialContestantData(element)) {
+            return
+        }
+
+        let status = null
+        if (element.col5){
+            status = element.col5 // help with navigating entry day wirdnes
+        } else {
+            status = element.col4
+        }
+        let teamName = element.name || element.name2
```

Comment:
Nit: Can we move this line below the if statement below? That way we can group the `status` code together

Reply:
Fixed in [this suggestion](https://github.com/jjm3x3/AmazingRaceFantasy/pull/125#discussion_r1677017323)